third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp

```diff
@@ -197,9 +200,14 @@ cl_int (CL_API_CALL *web_clGetGLTextureInfo)(cl_mem, cl_gl_texture_info, size_t,
 #define clEnqueueBarrierWithWaitList web_clEnqueueBarrierWithWaitList
 #define clCreateFromGLTexture web_clCreateFromGLTexture
 
-#if OS(LINUX) || OS(ANDROID)
+#if OS(ANDROID)
 #define MAP_FUNC(fn)  { *(void**)(&fn) = dlsym(handle, #fn); }
 #define MAP_FUNC_OR_BAIL(fn)  { *(void**)(&fn) = dlsym(handle, #fn); if(!fn) return false; }
+#else
+#define MAP_FUNC(fn) ASSERT_NOT_REACHED();
```

Comment:
On !Android MAP_FUNC() and its friends should never be reached. init() should return immediately after getCLHandle().